tests/Concerns/WithChunkReadingTest.php

```diff
@@ -522,4 +523,37 @@ public function registerEvents(): array
 
         $this->assertTrue($import->failed, 'ImportFailed event was not called.');
     }
+
+    /**
+     * @test
+     */
+    public function can_import_to_array_and_format_in_chunks()
```

Comment:
I think this test passes with and without your change? Could you update the test to have an example that would actually fail without your fix?

Reply:
If the code of is reverted as in the main branch (see [ReadChunk#L146](https://github.com/SpartnerNL/Laravel-Excel/blob/4fca093fbdcb8f4a3ce4870d8848dda4653af5f5/src/Jobs/ReadChunk.php#L146)) the test will fail.

Output:
```
There was 1 failure:
1) Maatwebsite\Excel\Tests\Concerns\WithChunkReadingTest::can_import_to_array_and_format_in_chunks
Failed asserting that 44896 is of type "string".
```
See also _describe your issue_ under #3832 why the code was changed. This assures the same behavior for normal readers and chunk readers.

Reply:
I see. However I think it would be good to assert the actual value instead of just checking it is a string no? If it returns `01-12-2022` or `'44896'` makes a difference